generators/server/templates/build.gradle.ejs

```diff
@@ -89,19 +89,14 @@ description = ''
 
 bootWar {
    mainClassName = '<%= packageName %>.<%= mainClass %>'
-}
-
-war {
```

Comment:
Currently we can generate the normal executable `.war` and a normal `.war.original` so maybe let's keep it for now until we have changed the default packaging to `.jar`.

Reply:
@atomfrede Good catch :+1:  Sorry about that but I missed it completely. I have pushed another commit that restores the generation of the secondary non-executable war. The difference is that instead of using `classifier` and relying on copy and rename, etc. I have instead used `extension = war.original` which actually has the same effect.
**NOTE:** Starting with Gradle 5.1 `extension` has been deprecated and replaced by  `archiveExtension`